kubernetes.go

```diff
@@ -157,11 +163,17 @@ func endpointHostnameIndexFunc(obj interface{}) ([]string, error) {
 	return hostnames, nil
 }
 
-func fetchEndpointIPs(endpoints []*endpoint.Endpoint, host string) (results []net.IP, ttl endpoint.TTL) {
+func fetchEndpointIPs(endpoints []*endpoint.Endpoint, host string, ip net.IP) (results []net.IP, ttl endpoint.TTL) {
 	for _, ep := range endpoints {
 		if ep.DNSName == host {
-			results = extractEndpointIPs(ep)
 			ttl = ep.RecordTTL
+			if ep.Labels["strategy"] == "geoip" {
```

Comment:
as it is going to be more strategies `switch` is more semantically compatible than `if`
https://gobyexample.com/switch

Reply:
@ytsarev it can be refactored later, and I believe linter would scream on using switch for 1 case, it would suggest to use if condition :)